Netable/NetableExample/ViewController/RootTabBarController.swift

```diff
@@ -13,8 +13,6 @@ class RootTabBarController: UITabBarController {
     override func viewDidLoad() {
         super.viewDidLoad()
 
-        UserRepository.shared.netable.requestFailureDelegate = self
```

Comment:
I think removing this but not adding it to the configuration in the sample app means this feature isn't getting tested any more, which is problem not ideal.

Reply:
Sorry, still working on some changes. I'll reassign when I think we're in a spot to review again